### PR TITLE
Added span.cpp for Span Problem in Others folder

### PR DESCRIPTION
**Pull request by mohitmo**
This PR is for adding span problem under others folder.

### CODE REVIEW

File: others/span.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            s.push(i);
+        }
+   }
+   
+}
+
+//testing our span function
+int main(){
+    //test-1
+    cout<<"TES-1\n";
```
Comment (cclauss) [inline others/span.cpp]: Is this missing a __T__?

File: others/span.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// Span Problem 
+// Here we have to find span of every element in an array, i.e number of consecutive elements before that element (including that element also)
+// having values less than or equal to value of current element
+// Note: Span of every element will be atleast 1
```
Comment (cclauss) [inline others/span.cpp]: __at least__ is two separate words.

Comment (mohitmo) [reply others/span.cpp]: Changes done as pointed by you

**Comment by cclauss** [comment]
https://github.com/TheAlgorithms/C-Plus-Plus/pull/493/checks?check_run_id=354706143#step:9:47

**Pull request closed without merging**